specification/07-theory_operations.adoc

```diff
@@ -81,10 +81,16 @@ I/O rules apply:
   used to protect the PCIe link between the TVM and the TDI.
 * A single selective IDE stream is established between the physical device and
   its Root Port. All TDIs from the physical device share this single IDE stream.
-* For each physical device from which a TDI is attached to a TVM, the TSM
-  generates, owns and distributes the IDE stream keys to both the physical
-  device and the RP.
-* The TSM configures the RP PCIe IDE extended capability.
+* The TSM that establishes the SPDM connection with a physical device must
+  generate, own and distribute the IDE stream keys to both the physical device
+  and the RP.
+* A TSM must not bind a TVM that it manages with a TDI from a physical device
+  for which it does own the IDE keys. In other words, TDIs can only be bound to
```

Comment:
Did you mean “does not own”? 
this sentence reads weird

Reply:
Yes, I meant `does not own`. Does that read better now?